### PR TITLE
message-error update and popup updates 

### PR DESCRIPTION
**Pull request by psnsVGC**
Dark Mode message-error update:http://prntscr.com/9vqw0i
Dark Mode popupmenu's update: http://prntscr.com/9vrh76 and http://prntscr.com/9vr3fj
Normal Mode popupmenu's update: http://prntscr.com/9vraxe and http://prntscr.com/9vr9ys

### CODE REVIEW

**Comment by AustinXII** [comment]
makes sure you squash @psnsVGC

**Comment by psnsVGC** [comment]
@AustinXII Ahh sorry made a big mistake last night by deleting the branch so had to restart

**Comment by Zarel** [comment]
This is the third pull request you've opened, after I already told you not to do this.

This is your last warning. I will ban you from the repository the next time you open a dark mode pullreq.

We are more than willing to teach you how to update an existing pullreq with rebasing and forcepushing, but you can't just ignore us like this.

**Comment by psnsVGC** [comment]
@Zarel oh sorry I never got warned must have been afk and yeah I would love to learn n_n

**Comment by Zarel** [comment]
I said right here: https://github.com/Zarel/Pokemon-Showdown-Client/pull/629#issuecomment-175961698

> Also please don't make multiple dark mode pullreqs; a single one is enough.

**Comment by AustinXII** [comment]
@psnsVGC  the warnings were on the other PRs, there is a rebasing guide in the roomintro in the dev room

**Comment by panpawn** [comment]
While rebasing, reminder to be more specific for a commit name.  Again: <b>if you do not understand how to rebase, ask instead of making a new pull request</b>

File: style/client.css [hunk 1]
```diff
@@ -3189,6 +3188,21 @@ a.ilink.yours {
...
+	position: absolute;
+	top: auto; right: auto; left: auto; bottom: auto;
+	text-align: left;
+	background: #0D0E0E;
+	color: #DDDDDD;
+	border: 1px solid #DDDDDD;
+	padding: 2px 10px;
+
+	border-radius: 6px;
+	box-shadow: inset 1px 1px 0 #ffffff, inset -1px -1px 0 #DDDDDD, 2px 2px 3px rgba(0,0,0,.2);
```
Comment (panpawn) [inline style/client.css]: ``0.2``

Comment (psnsVGC) [inline style/client.css]: @panpawn Your right I should have asked and will in the future and thanks for the edit never noticed that n_n

Comment (AustinXII) [inline style/client.css]: instead of ``.2`` you need to put ``0.2``

**Comment by psnsVGC** [comment]
@Slayer95 Thanks for reopening as I never closed this (unless missclicked but I doubt it) and thanks @AustinXII  I'll go change that n_n

**Comment by Zarel** [comment]
These changes have been superseded by my own dark mode improvements.

**Pull request closed without merging**